examples/bpi-monolith-test-suite/bpi/components/messaging/messageTypes.ts

```diff
@@ -0,0 +1,2 @@
+export const MESSAGE_TYPE_INFO = "INFO"; // Inform the recipient without changing the agreement state (i.e. Invitation)
```

Comment:
How about we have enums for messagetypes?

Reply:
I would go there in the monolith, as it only serves as a quick way to build a simulation of a BRI. Once we start translating it into a full blown BRI, enums are definitely better option than string constants

Reply:
Okay.